CIP-1694/README.md

```diff
@@ -0,0 +1,1142 @@
+
+Governance-related metadata can be clearly identified by registering a CIP-10 metadata label.
+Within that, the structure of the metadata can be determined by this CIP (exact format TBD), using an index to map the vote or governance action ID to the corresponding metadata URL and hash.
+
+This avoids the need to add additional fields to the transaction body, at the risk of making it easier for submitters to ignore.
+However, since the required metadata can be empty (or can point to a non-resolving URL),
+it is already easy for submitters to not provide metadata, and so it is unclear whether this makes the situation worse.
+
+Note that transaction metadata is never stored in the ledger state, so it would be up to clients
+to pair the metadata with the actions and votes in this alternative, and would not be available
+as a ledger state query.
+
+### Controlling the number of active governance actions
+
+Since governance actions are available for anyone to submit, we need some mechanism to prevent those
+individuals responsible for voting from becoming overwhelmed with a flood of proposals.
+A large deposit is one such mechanism, but this comes at the unfortunate cost of being a barrier
+for some people to submit an action.
+Note, however, that crowd-sourcing with a Plutus script is always an option to gather the deposit.
+
+We could, alternatively, accept the possibility of a large number of actions active at any given
+time, and instead depend on off-chain socialization to guide voters' attention to those that merit it.
+In this scenario, the constitutional committee might choose to only consider proposals which have
+already garnered enough votes from the DReps.
+
+### No AVST
+
+An earlier draft of this CIP included the notion of an "active voting stake threshold", or AVST.
+The purpose of AVST was to ensure the legitimacy of each vote, removing the possibility that, for example,
+9 out of 10 Lovelace could decide the fate of millions of entities on Cardano.
+There are really two concerns here, which are worth separating.
+
+The first concern is that of bootstrapping the system, i.e. reaching the initial moment when
+sufficient stake is registered to vote.
+The second concern is that the system could lose participation over time.
+One problem with the AVST is that it gives an incentive for SPOs to desire a low voting registration
+(since their votes then hold more weight).
+This is absolutely not a slight on the existing SPOs, but an issue with bad incentives.
+
+We have chosen, therefore, to solve the two concerns differently.
+We solve the bootstrapping problem as described in the section on bootstrapping.
+We solve the long-term participation problem by not allowing reward withdrawals
+(after the bootstrap phase) unless the stake is delegated to a DRep
+(including the two special cases, namely 'Abstain' and 'No confidence').
```

Comment:
This edit repackages the Rationale section into the desired properties of Minimum Viable Governance (MVG), with the goal of justifying the design choices made in CIP1694. The content of the old Rationale section is left mostly unchanged, it is simply presented differently. This is a rough draft and polishing ideas are welcome.

This structure clearly explains the intention behind every aspect of CIP1694, and highlights the flaws of certain ideas when lack evidence that they achieve these goals

Reply:
Once this new Rationale section is polished, I also recommend to move it towards the beginning of the document, before the more in-depth details of each solution

Reply:
There are a fair amount of authors on this CIP, and many of them have thoughts and opinions on how to order things, so big diffs like this are going to be extremely hard for me to manage. I really truly appreciate you taking the time and effort to actually write things down, and I'll do my best to have a look and see what I can add, but I just want to set the expectation that me as a human who is trying to manage a document like this with so many competing opinions is hard and I will perhaps not be able to keep up. :sweat: I'm about to push a bunch of re-words from Kevin (no real substantive change), and then taking this could be hard. Git is much better with code than prose.  Small concise edits might be easier for me in the future. Again, I don't want to sound unappreciative of you taking the initiative.

Reply:
thanks for the reply, I know it's probably not possible to straight up merge my version. I mostly wanted to show a real example of what I mean with "defining the problem", only using the existing content to show it's simply a matter of presentation, instead of simply commenting about it.

I'm happy if it makes other authors consider a more "scientific paper" presentation (not necessarily exactly mine) after seeing my example. It would definitely help the discourse around this CIP in the future ;)